askama/src/lib.rs

```diff
@@ -73,7 +73,8 @@ pub use askama_shared::{
 pub mod mime {
     #[cfg(all(feature = "mime_guess", feature = "mime"))]
     #[deprecated(since = "0.11.1", note = "Use Template::MIME_TYPE instead")]
-    pub use crate::shared::extension_to_mime_type;
+    #[allow(non_upper_case_globals)]
+    pub const extension_to_mime_type: () = ();
```

Comment:
What's the point of this? It's the same symbol, but it has a different type.

Reply:
To keep the deprecation notice, so the user gets a message how to proceed even thought their compilation will fail.

Reply:
It's an interesting idea, but I think we should just get rid of the symbol instead. Probably not a lot of people using it anyway.